README.md

```diff
@@ -1,2 +1,20 @@
 # hw01
-Repository for homework 1
+
+![picture](PictureofMarie.jpg)
```

Comment:
Great job. The [picture] in brackets actually does not appear by default; there are other options for adding a caption or scroll over descriptive text.